src/libcrun/utils.c

```diff
@@ -1930,7 +1930,7 @@ find_executable (const char *executable_path, const char *cwd)
         it = ".";
 
       len = snprintf (path, PATH_MAX, "%s/%s", it, executable_path);
-      if (len == PATH_MAX)
```

Comment:
Is it possible for len to be  > PATH_MAX?  snprintf would ensure it is not, correct?

Reply:
Yes, at least that is how I understand the man page 
 https://man7.org/linux/man-pages/man3/fprintf.3p.html#RETURN_VALUE

This PR matches this example:
https://github.com/containers/crun/blob/056a40764b935c343ac8c58118392ebdbc5e4369/src/libcrun/utils.h#L458